app/transport/http/server.go

```diff
@@ -12,35 +12,34 @@ import (
 
 const timeoutMsg = "Connection timeout"
 
-// Server is simple server
+// Server is simple server.
 type Server struct{ *http.Server }
 
 type Handlers struct {
 	UserHandler UserHandler
 }
 
-// NewServer will initialize the server
+// NewServer will initialize the server.
 func NewServer(opt cfg.Options, handlers Handlers, logger *lgr.Logger) *Server {
 	router := new(mux.Router)
 
 	attachUserEndpoints(router, handlers)
 
 	handler := WrapMiddleware(router,
 		WithLogRequest(logger),
-		// to be continue...
 	)
 
 	return &Server{
 		Server: &http.Server{
 			Addr:         opt.Server.Host + ":" + opt.Server.Port,
-			Handler:      http.TimeoutHandler(handler, 3*time.Second, timeoutMsg),
-			ReadTimeout:  2 * time.Second,
-			WriteTimeout: 5 * time.Second,
+			Handler:      http.TimeoutHandler(handler, 3*time.Second, timeoutMsg), //nolint:gomnd //2 is not magic number
```

Comment:
You left a comment that 2 is not a magic number but it is. 
Pls make constants for read... write... timeouts and use them.

Reply:
done